README.md

```diff
@@ -1,96 +1,155 @@
+  // The values below are found on the Application settings tab.
+  'domain'        => '{YOUR_TENANT}.auth0.com',
+  'client_id'     => '{YOUR_APPLICATION_CLIENT_ID}',
+  'client_secret' => '{YOUR_APPLICATION_CLIENT_SECRET}',
+
+  // This is your application URL that will be used to process the login.
+  // Save this URL in the "Allowed Callback URLs" field on the Application settings tab
+  'redirect_uri' => 'https://{YOUR_APPLICATION_CALLBACK_URL}',
 ]);
 ```
 
-**The values above should not be hard-coded in a production application** but will suffice for testing or local development. Please see our complete guide on the [main documentation page](https://auth0.com/docs/libraries/auth0-php#getting-started) for more information on how to store and use these values.
+**Note:** _In a production application you should never hardcode these values. Consider using environment variables to store and pass these values to your application, as suggested in our [documentation](https://auth0.com/docs/libraries/auth0-php#getting-started)._
+
+Using the SDK, making requests to Auth0's endpoints couldn't be simpler. For example, signing users in using Auth0's [Universal Login](https://auth0.com/docs/universal-login) and retrieving user details can be done in a few lines of code:
+
+```php
+// Do we have an authenticated session available?
+if ($userinfo = $auth0->getUser()) {
+  // Output the authenticated user
+  var_dump($userinfo);
+  exit;
+}
+
+// No session was available, so redirect to Universal Login page
+$auth0->login();
+```
+
+Further examples of how you can use the Authentication API Client can be found on [our documentation site](https://auth0.com/docs/libraries/auth0-php/).
+
+### Management API Client
+
+This SDK also offers an interface for Auth0's Management API, using a management access token.
+
+Management access tokens can be generated using the `getManagementAccessToken` method and the Authentication API, please review our [documentation](https://auth0.com/docs/libraries/auth0-php/using-the-management-api-with-auth0-php) for guidance.
+
+```php
+use Auth0\SDK\API\Management;
+
+$mgmt_api = new Management('{{YOUR_MANAGEMENT_ACCESS_TOKEN}}', 'https://{{YOUR_TENANT}}.auth0.com');
+```
+
+The SDK provides convenient interfaces to the Management API's endpoints. For example, to search for users:
+
+```php
+$results = $mgmt_api->users()->getAll([
+  'q' => 'josh'
+]);
+
+if (! empty($results)) {
+  echo '<h2>User Search</h2>';
+
+  foreach ($results as $datum) {
+    printf(
+      '<p><strong>%s</strong> &lt;%s&gt; - %s</p>',
+      !empty($datum['nickname']) ? $datum['nickname'] : 'No nickname',
+      !empty($datum['email']) ? $datum['email'] : 'No email',
+      $datum['user_id']
+    );
+  }
+}
+```
+
+At the moment the best way to see what endpoints are covered is to read through the `\Auth0\SDK\API\Management` class, [available here](https://github.com/auth0/auth0-PHP/blob/update-readme/src/API/Management.php).
```

Comment:
We could use

> At the moment, the best way to see what endpoints are covered is to read through the [`\Auth0\SDK\API\Management`](https://github.com/auth0/auth0-PHP/blob/update-readme/src/API/Management.php) class.

We will also need to update the URL as it points to the update-readme branch instead of master.

Reply:
Good catch